packages/client-analytics/src/types/GetABTestsOptions.ts

```diff
@@ -8,4 +8,16 @@ export type GetABTestsOptions = {
    *  The limit of the number of ab tests returned.
    */
   readonly limit?: number;
+
+  /**
+   *  Filters the returned ab tests by any indices starting with the
+   *  provided prefix that are assigned to either variant of an ab test.
+   */
+  readonly indexPrefix?: string;
+
+  /**
+   *  Filters the returned ab tests by any indices ending with the
+   *  provided suffix that are assigned to either variant of an ab test.
+   */
+  readonly indexSuffix?: string;
```

Comment:
can they both be defined? If not we should either have a union type or at least the description mentioning it

Reply:
yes they can both be defined.